source/draw/gpu/opengl/backend/gles3/Gles3IndexBufferObject.ooc

```diff
@@ -0,0 +1,57 @@
+/* This file is part of magic-sdk, an sdk for the open source programming language magic.
+ *
+ * Copyright (C) 2016 magic-lang
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+use base
+use geometry
+import external/gles3
+import Gles3Debug
+import Gles3VertexBufferObject
+
+version(!gpuOff) {
+Gles3IndexBufferObject: class {
+	_backend: UInt
+	_vbo: Gles3VertexBufferObject
+	_indexCount: Int
+	init: func (vertices: FloatPoint3D[], textureCoordinates: FloatPoint2D[], indices: IntPoint3D[]) {
+		this _indexCount = 3 * indices length
```

Comment:
`indices length` equals the number of `IntPoint3D` which equals `indices length * 3` total indices.